__tests__/app.test.js

```diff
@@ -106,7 +106,49 @@ describe ('GET /api/articles/:article_id',() => {
                 expect(article).toHaveProperty("votes", expect.any(Number));   
                 expect(article).toHaveProperty("article_img_url", expect.any(String));
                 expect(article).toHaveProperty("comment_count", expect.any(Number));
+                expect(article).not.toHaveProperty("body"); 
             })        
         })
       }); 
+  })
+
+  //Task 6 GET /api/articles/:article_id/comments
+describe ('GET /api/articles/:article_id/comments',() => {
+    test ('200:should return comments by article_id',() => {
+    return request(app)
+    .get('/api/articles/1/comments')
+    .expect(200)
+    .then (({body}) => {
+        const {comments} = body;
+        expect(comments).toBeSortedBy("created_at",{descending: true,});
+        comments.forEach((comment) => {
+          expect(comment).toHaveProperty("comment_id", expect.any(Number));
+          expect(comment).toHaveProperty("votes", expect.any(Number));   
+          expect(comment).toHaveProperty("created_at");    
+          expect(comment).toHaveProperty("author", expect.any(String));
+          expect(comment).toHaveProperty("body", expect.any(String));
+          expect(comment).toHaveProperty("article_id", expect.any(Number));
```

Comment:
we know the article id on all of these comments will be 1 because we're only getting the comments for article 1

Reply:
Added  in another 200 test for if the GET request is successful but there are no comments for that article